src/app/models/message.ts

```diff
@@ -1,9 +1,9 @@
-import { firestore } from 'firebase';
-
+//import { firestore } from 'firebase';
 export interface MessageI {
   id?: string;
```

Comment:
Notice this id has a question mark, indicating that it is optional. This was so that we could create instances of it without the id, and a use case is some of the changes I am suggesting.

Reply:
Not sure what to change. 
//import { firestore } from 'firebase';
was on line1.

Reply:
Not suggesting to change anything in this file. Just pointing out something about TypeScript interfaces and optional properties.